### PR TITLE
Post author about

### LINKED ISSUE #545: About author
Add the about author section under single blogs posts views

> [Design](https://www.figma.com/file/iZtsffS8kRYeIG5xrhbp1O/Subvisual-Blog-(Client)?type=design&node-id=1-2100&t=M4amWVpi3qHx3a43-4)

### PR DESCRIPTION
**Pull request by jreis13**
Why:
- Closes #545 

These changes address the need by:
- Update schema
- Add `<PostAuthorAbout />` & styles
- Adjust spacings to match design

### CODE REVIEW

File: config/node/createSchemaCustomization.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ module.exports = ({ actions }) => {
         initials: String,
         email: String,
         photo: Photo
+        bio: String,
```
Comment (mirhamasala) [inline config/node/createSchemaCustomization.js:12]: @gabrielpoca Can you confirm the schema stuff? Here ... and ...

File: src/templates/Post/index.jsx [hunk 2]
```diff
@@ -24,6 +24,7 @@ export const query = graphql`
           key
           name
           initials
+          bio
```
Comment (mirhamasala) [inline src/templates/Post/index.jsx:27]: @gabrielpoca Here?

**Review by mirhamasala — APPROVED** [review]

File: src/components/PostAuthorAbout/index.jsx [hunk 3]
```diff
@@ -7,10 +7,10 @@ import * as styles from "./index.module.scss"
 function PostAuthorAbout({ author, date }) {
   return (
     <div className={styles.root}>
-      <div className={styles.author}>
+      <div className={styles.header}>
         <Author author={author} date={date} />
       </div>
-      <p className={styles.about}>About the author</p>
+      <p className={styles.heading}>About the author</p>
       <p className={styles.bio}>{author.bio}</p>
```
Comment (mirhamasala) [inline src/components/PostAuthorAbout/index.jsx:14]: Does this css class exist? `bio`?

File: src/templates/Post/index.jsx (hunk 2 above)
Comment (gabrielpoca) [reply src/templates/Post/index.jsx:27]: 👍

**Pull request merged**